scripts/run-integration-tests-npm.sh

```diff
@@ -297,8 +297,8 @@ fi
 
 # --- Running Mock Blockchain --- {{{
 log "Running hardhat local node"
-HOPR_ENVIRONMENT_ID="hardhat-localhost" yarn workspace @hoprnet/hopr-ethereum hardhat node \
-  --network hardhat --show-stack-traces > \
+HOPR_ENVIRONMENT_ID="hardhat-localhost" TS_NODE_PROJECT=${mydir}/../packages/ethereum/tsconfig.hardhat.json yarn workspace @hoprnet/hopr-ethereum run network \
```

Comment:
The `TS_NODE_PROJECT` is already set in the yarn script.

Reply:
changed